charts/rancher-monitoring/106.0.0+up61.3.2/Chart.yaml

```diff
@@ -14,7 +14,7 @@ annotations:
   catalog.cattle.io/namespace: cattle-monitoring-system
   catalog.cattle.io/permits-os: linux,windows
   catalog.cattle.io/provides-gvr: monitoring.coreos.com.prometheus/v1
-  catalog.cattle.io/rancher-version: '>= 2.10.0-0 < 2.11.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.11.0-0 < 2.12.0-0'
```

Comment:
Hello @alexandreLamarre per the [review](https://github.com/rancher/charts/pull/5064#pullrequestreview-2591329415) on #5064 I have incorporated the rancher version bumps too in the same [commit](https://github.com/rancher/charts/pull/5064/commits/8156f65cce1505f3405128368e082b541904d428).
Would it be feasible to eliminate this rancher version bump changes from here, and continue with merging of #5064?
cc @jbiers @mallardduck